### DISCUSSION TITLE
ids of minisign pubkeys differ

### DISCUSSION THREAD

**Discussion by ghburdie** [DNSCrypt/dnscrypt-proxy#2111] [Potential issues]
# Are the download files corrupted? or ...
## Did you change your private key and are still publishing the old pubkey?
After Downloading [dnscrypt-proxy-win64-2.1.1.zip](https://github.com/DNSCrypt/dnscrypt-proxy/releases/download/2.1.1/dnscrypt-proxy-win64-2.1.1.zip), [dnscrypt-proxy-win64-2.1.1.zip.minisig](https://github.com/DNSCrypt/dnscrypt-proxy/releases/download/2.1.1/dnscrypt-proxy-win64-2.1.1.zip.minisig) and using pubkey of file [jedi-minisign.pub](https://sk.tl/7CPRo8kn) and executing:

`minisign -VP RWQf6LRCGA9i53mlYecO4IzT51TGPpvWucNSCh1CBM0QTaLn73Y7GFO3 -m dnscrypt-proxy-win64-2.1.1.zip`

I get following result:
`
Signature key id in dnscrypt-proxy-win64-2.1.1.zip.minisig is 79833371EA15D7E4
but the key id in the public key is E7620F1842B4E81F
`

**Comment by jedisct1** [comment]
See the very first page of the documentation: https://github.com/DNSCrypt/dnscrypt-proxy/wiki/Installation#verification-of-downloaded-files

**Reply by ghburdie** [reply]
That worked.
Thank you!
